collections/_release_4_5/entry-general-editor-animation-player-quality-of-life-updates.md

```diff
@@ -0,0 +1,22 @@
+---
+type: entry
+section: general
+subsection: editor
+rank: 10
+importance: 2
+anchor: animation-player-quality-of-life-upgrades
+title: Animation player quality-of-life upgrades
+text: |
+  The animation player has received some needed love!
+  - You can now use the selection box to select and scale Bézier editor points. Now it’s a piece of cake to perform batch changes.
+  - When creating Bézier points, you can now auto-tangent new points in a balanced or mirrored manner.
+  - You can now sort the animation player’s animations alphabetically.
+  - It is also now possible to filter animations by their name!
```

Comment:
I wonder if we don't need so many "now"s, which get a bit grating after so many in a row? In general, everything in these release notes is new, i.e. something you can now do. So maybe just:
```suggestion
  - You can use the selection box to select and scale Bézier editor points. Now it’s a piece of cake to perform batch changes.
  - When creating Bézier points, you can auto-tangent new points in a balanced or mirrored manner.
  - You can sort the animation player’s animations alphabetically.
  - It is also possible to filter animations by their name!
```

Reply:
That sounds better!